samples/nrf9160/nrf_cloud_mqtt_multi_service/src/connection.h

```diff
@@ -7,6 +7,23 @@
 #define _CONNECTION_H_
 #include <cJSON.h>
 
+/**
+ * @brief Callback type for receiving device messages from nRF Cloud.
```

Comment:
Can this be just: "Receive device messages from ..."?
See https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#functions

Reply:
It could be "nRF Cloud device message handler" if adopting the phrasing in that example

I'll make that change